cognite/client/_api_client.py

```diff
@@ -795,6 +796,8 @@ def _create_multiple(
         else:
             items = cast(Union[Sequence[T_CogniteResource], Sequence[Dict[str, Any]]], items)
 
+        items = [item.as_write() if isinstance(item, WriteableCogniteResource) else item for item in items]
```

Comment:
This will always be the case, no? Can update the signature of the method instead.
```suggestion
        items = [item.as_write() for item in items]
```

Reply:
I updated the signature, but since the functions accepts items of `dict` we need the if check.